src/Instrumentation/Psr15/README.md

```diff
@@ -0,0 +1,26 @@
+# OpenTelemetry PSR-15 auto-instrumentation
```

Comment:
*Just wanna say I'm super excited for this* :fire:

Reply:
Thanks very much for the review feedback. I agree with all and have addressed them.